UnityProject/Assets/Scripts/Lifecycle/PlayerSpawn.cs

```diff
@@ -23,6 +23,8 @@ public static GameObject ServerSpawnPlayer(JoinedViewer joinedViewer, Occupation
 	{
 		NetworkConnection conn = joinedViewer.connectionToClient;
 
+		if (conn == null) return null;
```

Comment:
This one is more complicated than just null checking to fix. As we need to find out why it null, NoooneyDude is looking into it.

Reply:
what do you say about merging?

Reply:
Guess you can merge it, who knows maybe it'll fix it.

Reply:
> This one is more complicated than just null checking to fix. As we need to find out why it null, NoooneyDude is looking into it. Its stopping players from spawning in.

So, I'll remove this line in next commit

Reply:
Oh ok, up to you.